quirks/password-rules.json

```diff
@@ -53,6 +53,15 @@
     "comcastpaymentcenter.com": {
         "password-rules": "minlength: 8; maxlength: 20; required: lower, upper; required: digit; max-consecutive: 2;"
     },
+    "commerzbank.de": {
+        "password-rules": "required: upper,lower; required: digit; minlength: 5; maxlength: 8;"
+    },
+    "consorsbank.de": {
+        "password-rules": "required: upper,lower, digit; minlength: 5; maxlength: 5;"
```

Comment:
5 character passwords? Is this more of a kind of PIN?

Reply:
That's normal with German banks, you have a randomly chosen password that is rather short e.g. 5 chars long. You can call it a PIN if you like. For example Sparkasse (one of the biggest German banks) calls it "PIN" in the login form. However, it's the secret that you need to sign into your bank account.